lib/galaxy/tools/parameters/basic.py

```diff
@@ -151,7 +151,7 @@ def value_to_display_text( self, value, app ):
         Convert a value to a text representation suitable for displaying to
         the user
         """
-        if value:
+        if value is not None:
             return unicodify( value )
         return "Not available."
```

Comment:
I thought about overriding it and providing e.g. `Empty string` or `""`, etc, but all of those are actually valid values for text inputs as well. Perhaps if it was instead *italicized*, along with the message, that would work.